sql/src/main/java/io/crate/analyze/relations/RelationAnalyzer.java

```diff
@@ -488,6 +472,81 @@ private static Symbol symbolFromSelectOutputReferenceOrExpression(Expression exp
         return symbol;
     }
 
+    private List<Symbol> analyzeGroupBy(SelectAnalysis selectAnalysis,
```

Comment:
is this re-ordering of the method necessary? Makes it difficult to see any actual change in the diff